backend-rust/src/graphql_api/baker.rs

```diff
@@ -498,7 +498,7 @@ impl Baker {
             // u128.
             let capital_bound_cap_for_pool_numerator = capital_bound
                 * ((total_stake - delegated_stake_of_pool) as u128)
-                - (100_000 * (self.staked as u128));
+                    .saturating_sub(100_000 * (self.staked as u128));
```

Comment:
From what I could see, staked should not be multiplied by 100_000 based on the state the nodes which I queried was in.

This could be related to the genesis loading.

Please add a todo comment

Reply:
For lower protocol version the formula is off since `leverage_bound/capital_bound` does not exist.

Reply:
It exists using the current database in stagenet querying baker 0-4.